pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -93,6 +93,50 @@ def add_oa_cuts(target_model, dual_values, solve_data, config,
                           >= constr.lower)
                 )
 
+def add_ecp_cuts(target_model, solve_data, config,
+                linearize_active=True,
+                linearize_violated=True,
+                linearize_inactive=False):
+    """Linearizes nonlinear constraints.
+
```

Comment:
Please complete the documentation here to say that it adds the cuts for the ECP method.